README.adoc

```diff
@@ -40,20 +40,21 @@ Open Liberty. If you run the application, Maven creates an Open Liberty server a
 the application on it. The application displays a simple web page with a link that, when 
 clicked, calls the servlet to return a simple response of `Hello! How are you today?`.
 
-One benefit of using Maven is that, once you have defined the details of the project and 
-any dependencies it has, Maven automatically handles downloading and installing any and 
-all of the dependencies.
+One benefit of using a build tool like Maven is that you can define the details of the project and any dependencies it has, and then Maven automatically downloads and installs the dependencies.
+Another benefit of using Maven is that it can run repeatable, automated tests on the application. 
+You could, of course, test your application manually by starting a server and pointing a web browser at the application URL. 
+Automated tests are a much better approach because you can easily rerun the same tests each time the application is built. 
```

Comment:
Suggested update

>However, automated tests are a much better approach because you can easily rerun the same tests each time the application is built.